core/controllers/async_download.py

```diff
@@ -147,7 +147,11 @@ def get_presigned_url(filename: str):
 
     url = _S3_CLIENT.generate_presigned_url(
         "get_object",
-        Params={"Bucket": Config.AWS_S3_BUCKET_FIND_DATA_FILES, "Key": filename},
+        Params={
+            "Bucket": Config.AWS_S3_BUCKET_FIND_DATA_FILES,
+            "Key": filename,
+            "ResponseContentDisposition": f'attachment; filename="{filename}"',
+        },
         ExpiresIn=1600,
```

Comment:
I think now we can make it to expire in like 60 seconds as you said?

Reply:
We can do that in a separate PR - might as well keep this small and focused :) But yes - we can do that. Do you want to make a PR for it?

Reply:
Sure, I'll raise one PR for it.